bot/storage-mongo/src/main/kotlin/UserTimelineMongoDAO.kt

```diff
@@ -247,63 +244,62 @@ internal object UserTimelineMongoDAO : UserTimelineDAO, UserReportDAO, DialogRep
             logger.debug { "dialog saved $userTimeline" }
         }
 
-        executor.executeBlocking {
```

Comment:
This is not a good idea to remove this executeBlocking block. Why doing this ?

Reply:
Due to data state synchronization issue in coroutines.

Now it's fixed. I implemented the following solution: https://kotlinlang.org/docs/shared-mutable-state-and-concurrency.html#thread-confinement-coarse-grained
I

Reply:
I still don't like the PR.. I suggest we start a call about this :)

Reply:
Yes, I'm available for the rest of the week. We can talk about this RP.
What about you?

Reply:
I'v sent a message on gitter ;)

Reply:
Following our discussion, the code has been updated